fitness/data/appointment_view.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <odoo>
     <template id="custom_appointment_info_view" inherit_id="appointment.appointment_info">
-        <xpath expr="//div[@t-if=&quot;appointment_type.category in ['punctual', 'recurring']&quot;]" position="replace">
-            <div t-if="slots and not based_on_users and appointment_type.resource_manage_capacity and max_capacity" class="mb-4">
+        <xpath expr="//div[@t-if=&quot;not based_on_users and appointment_type.resource_manage_capacity&quot;]" position="replace">
+            <div t-if="slots and max_capacity" class="mb-4">
```

Comment:
This looks doubtful + appointments should not be free if not loggedin

Reply:
@jaeschwa can you have a look here please? :pray: 
Base view changed, so custo needs to as well